### PR TITLE
Replace dials with graphs on cluster stats for cpu/mem

### PR DESCRIPTION
**Pull request by hsmade**
WIP/POC

### CODE REVIEW

File: frontend/src/reducers/root.js [hunk 1]
```diff
@@ -3,10 +3,11 @@ import { combineReducers } from "redux"
 import {
   AppDrawer,
   AppErrorReducer,
-  ClusterStatisticsReducer,
+  // ClusterStatisticsReducer,
```
Comment (jippi) [inline frontend/src/reducers/root.js]: delete this ;)

File: frontend/src/reducers/utils.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
+import format from "date-fns/format"
+
+export function calculateNodeStats(state, payload, AmountDataPoints) {
```
Comment (jippi) [inline frontend/src/reducers/utils.js]: in JS i would call this `amountDataPoints` -  or maybe `maxLength`?

Comment (hsmade) [reply frontend/src/reducers/utils.js]: I'll go with maxLength.

**Comment by hsmade** [comment]
https://snag.gy/otb1IU.jpg
https://snag.gy/tsJALT.jpg

**Comment by hsmade** [comment]
Change the chars to show actual figures (MHz + GB _and_ percentage)

**Pull request merged**